library/core/src/main/java/com/google/android/exoplayer2/drm/HttpMediaDrmCallback.java

```diff
@@ -47,29 +47,36 @@
   }
 
   private final HttpDataSource.Factory dataSourceFactory;
-  private final String defaultUrl;
+  private final String defaultLicenseUrl;
+  private final Boolean forceDefaultLicenseUrl;
   private final Map<String, String> keyRequestProperties;
 
   /**
-   * @param defaultUrl The default license URL.
+   * @param defaultLicenseUrl The default license URL.
+   * @param forceDefaultLicenseUrl If true, default license URL will be used in the license
+   *    acquisition process even do there is a license URL defined in the stream manifest.
```

Comment:
"stream manifest" is too specific to one of a number of possible use cases of this class. For the purpose of this class I think it should just read something like: "Whether to force use of {@code defaultLicenseUrl} even for key requests that include their own license URL."

nit: Indent should be 4 chars for a continuation of a comment, for consistency with rest of code.